node/forest_libp2p/src/chain_exchange/behaviour.rs

```diff
@@ -55,7 +55,7 @@ impl ChainExchangeBehaviour {
         if let Some(channel) = self.response_channels.remove(request_id) {
             self.track_metrics();
             if let Err(err) = channel.send_async(Ok(response)).await {
-                warn!("{err}");
+                debug!("{err}");
```

Comment:
Could you write a comment here explaining how the errors are expected and not actually errors.

Reply:
Fixed